src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -88,6 +88,40 @@ def test_self_cal(session):
     session.self_cal()
 
 
+def test_channels_rep_cap():
+    with nifgen.Session('', '', False, 'Simulate=1, DriverSetup=Model:5433 (2CH);BoardType:PXIe') as session:
+        session.func_amplitude = 0.5
+        assert session.channels['0-1'].func_amplitude == 0.5
+
+        session.channels[0].func_amplitude = 1
+        assert session.channels['0'].func_amplitude == 1
+        assert session.channels[1].func_amplitude == 0.5
+
+
+def test_markers_rep_cap(session):
```

Comment:
Should I just use verbose strings everywhere (as an example for how good code should be written) in lieu of coverage for more inputs to the repeated capability?
If so, I'll apply the answer to all of the new tests.